nodejs/eks/examples/tests/replace-cluster-add-subnets/index.ts

```diff
@@ -0,0 +1,90 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as eks from "@pulumi/eks";
+
+const projectName = pulumi.getProject();
+const zones = aws.getAvailabilityZones();
+const tags = { "Name": `${projectName}` };
+
+/**
+ * VPC
+ */
+const vpcCidr = "10.0.0.0/16";
```

Comment:
FWIW - this test could probably be massively simplified by using awsx.ec2.Vpc. That might help make it clearer what’s changing between runs, and what’s the essential point of the test.

Reply:
Even if you don’t love to that, it might be worth splitting this into two files, so that the file you replace in the second stendoesnt have to copy/paste quite so much.

Reply:
Great points. I'll make sure to simplify this by using `awsx` instead of raw `aws`

Reply:
Done